test/transpile.jl

```diff
@@ -888,37 +1034,59 @@ end
     ]
 
     for (input_gate, type_result) in test_inputs
-        circuit = QuantumCircuit(qubit_count = target, gates = [input_gate])
+        circuit = QuantumCircuit(qubit_count = target, instructions = [input_gate])
 
         transpiled_circuit = transpile(transpiler, circuit)
 
         @test compare_circuits(circuit, transpiled_circuit)
 
-        @test get_gate_symbol(get_circuit_gates(transpiled_circuit)[1]) isa type_result
+        @test get_gate_symbol(get_circuit_instructions(transpiled_circuit)[1]) isa
+              type_result
     end
 
-    circuit = QuantumCircuit(qubit_count = target, gates = [rotation_x(target, 0.0)])
+    circuit = QuantumCircuit(qubit_count = target, instructions = [rotation_x(target, 0.0)])
 
     transpiled_circuit = transpile(transpiler, circuit)
 
     @test compare_circuits(circuit, transpiled_circuit)
 
-    @test length(get_circuit_gates(transpiled_circuit)) == 0
+    @test length(get_circuit_instructions(transpiled_circuit)) == 0
 
     # with default tolerance
-    circuit = QuantumCircuit(qubit_count = target, gates = [rotation_x(target, 1e-3)])
+    circuit =
+        QuantumCircuit(qubit_count = target, instructions = [rotation_x(target, 1e-3)])
 
     transpiled_circuit = transpile(transpiler, circuit)
 
-    @test length(get_circuit_gates(transpiled_circuit)) == 1
+    @test length(get_circuit_instructions(transpiled_circuit)) == 1
 
     # with user-defined tolerance
 
     transpiler = SimplifyRxGatesTranspiler(1e-1)
 
     transpiled_circuit = transpile(transpiler, circuit)
 
-    @test length(get_circuit_gates(transpiled_circuit)) == 0
+    @test length(get_circuit_instructions(transpiled_circuit)) == 0
+end
+
+@testset "SimplifyRxGatesTranspiler: Readout" begin
+    transpiler = CastSwapToCZGateTranspiler()
```

Comment:
`transpiler` does not match the test name

Reply:
fixed, was supposed to test `SimplifyRxGatesTranspiler`